microshift_support/microshift-etcd.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="etcd-microshift"]
+= MicroShift etcd systemd log stream
+//another name idea could be "Etcd as a transient systemd unit that is managed by MicroShift"
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: etcd-microshift
+
+toc::[]
+
+MicroShift etcd logs will operate in their own system log stream by running the etcd container image under Podman and running the etcd binary installed on a MicroShift RPM.
```

Comment:
etcd does not run in a container, so podman is not used. It runs as a transient systemd unit, and that's what ensures the logs are separated. However, I don't think we want user documentation to go into that level of detail about implementation. I think it is sufficient to say something like "MicroShift etcd runs as a separate process and the logs will appear in their own system log stream called name-goes-here."